client/src/components/PageLayout/Header.tsx

```diff
@@ -84,27 +48,75 @@ export const Header: React.FC = () => {
             width="100%"
           />
         </Link>
-        {loadingUser ? (
-          <Spinner color="white" size="xl" />
-        ) : (
-          <HStack as="nav">
-            <Box>
-              <HeaderMenu
-                logout={logout}
-                goHome={goHome}
-                canAuthenticateWithGoogle={canAuthenticateWithGoogle}
-                user={user as MeQuery}
-                LoginButton={LoginButton}
-              />
-            </Box>
-            {user && (
-              <NextLink passHref href="/profile">
-                <Avatar user={user} cursor="pointer" />
-              </NextLink>
-            )}
-          </HStack>
-        )}
-      </HeaderItem>
+        <HStack as="nav">
+          <Box>
+            <Menu>
+              <MenuButton
+                as={Button}
+                aria-label="Options"
+                variant="outline"
+                background="gray.10"
+                px={[2, 4]}
+                py={[1, 2]}
+              >
+                Menu
+              </MenuButton>
+              <MenuList paddingBlock={0}>
+                <Flex
+                  flexDirection="column"
+                  fontWeight="600"
+                  borderRadius="5px"
+                >
+                  <NextLink passHref href="/chapters">
+                    <MenuItem as="a">Chapters</MenuItem>
+                  </NextLink>
+
+                  <NextLink passHref href="/events">
+                    <MenuItem as="a">Events</MenuItem>
+                  </NextLink>
+
+                  {user && (
+                    <Box borderBlock={'1px'} borderColor={'gray.85'}>
+                      <NextLink passHref href="/profile">
+                        <MenuItem as="a">Profile</MenuItem>
+                      </NextLink>
+                      {CanEditDashboard && (
+                        <NextLink passHref href="/dashboard/chapters">
+                          <MenuItem as="a">Dashboard</MenuItem>
+                        </NextLink>
+                      )}
+                    </Box>
+                  )}
+                  {user ? (
+                    <MenuItem
+                      data-cy="logout-button"
+                      onClick={() => logout().then(goHome)}
+                      fontWeight="600"
+                      height={'100%'}
+                    >
+                      logout
```

Comment:
```suggestion
                      Logout
```
To be consistent with the rest.